packages/main/src/Popup.ts

```diff
@@ -592,6 +628,30 @@ abstract class Popup extends UI5Element {
 		return getEffectiveAriaLabelText(this);
 	}
 
+	get _accInfoAriaDescription() {
+		return this.ariaDescriptionText || "";
+	}
+
+	get _accInfoAriaDescriptionId() {
```

Comment:
I don't think we have this "_accInfoAriaDescriptionId". This is something specific for the Input component, but not for the Popup.